low-level/files/opt/seagate/sspl/conf/sspl.conf.sample.yaml

```diff
@@ -139,7 +139,7 @@ NODEDATAMSGHANDLER:
 
 SYSTEMDWATCHDOG:
    threaded: true
-   monitored_services: 
```

Comment:
i see sample config file structure not in sync with LR2 latest config file. we will have to address that separately soon

Reply:
Okay